README.md

```diff
@@ -1,6 +1,45 @@
-# hypermodules
+<h1 align="center">✨<a href="https://github.com/hypermodules">hypermodules</a>⚡️</h1>
```

Comment:
i think it looks better with sparkles on both sides

### **✨ hypermodules ✨**

more magical